Snakefile

```diff
@@ -137,7 +137,7 @@ if USE_ENERGY_PLUS:
                 SECDIR_path,
                 "demand", 
                 "heat",    
-                "heat_demand_{demand}_s{simpl}_{clusters}_{planning_horizons}_ep.csv",
+                "heat_demand_{demand}_s{simpl}_{clusters}_{planning_horizons}.csv",
```

Comment:
@SermishaNarayana one more point: removing `_ep` (stays for `EnergyPlus` outputs) here means that `prepare_sector_network` will use outputs of the upstream workflow ("PyPSA-Eur" approach) not outputs of `aggregate_energyplus`. Is it intentional?